src/octonode/user.coffee

```diff
@@ -15,21 +15,29 @@ class User
     Object.keys(data).forEach (e) =>
       @[e] = data[e]
 
-  # Get a user
-  # '/users/pkumar' GET
+  # Get a user (promise)
   info: (cb) ->
-    @client.get "/users/#{@login}", (err, s, b, h)  ->
-      return cb(err) if err
-      if s isnt 200 then cb(new Error('User info error')) else cb null, b, h
```

Comment:
Just making sure you understand that there will be no more custom errors per API endpoint now.  This might not be acceptable for the maintainer